app/views/refinery/inquiries/admin/inquiries/show.html.erb

```diff
@@ -46,6 +46,16 @@
         <%= @inquiry.name %> [<%= mail_to @inquiry.email, @inquiry.email, {:title => t('.click_to_email')} %>]
       </td>
     </tr>
+    <% unless @inquiry.company.blank? %>
```

Comment:
you can use `if @inquiry.company.present?` (though I can see we use the `unless` style elsewhere too..)

Reply:
I changed this test to `if` as recommended, note that the other tests might also need changing for consistency.